app/controllers/guide_cards_controller.rb

```diff
@@ -8,6 +8,7 @@ def index
 
   def search
     @exact_match = GuideCard.find_by(heading: params[:search_term])
+    @exact_match = GuideCard.where('heading < ?', 'Aarons').limit(1).order(heading: :desc).first if @exact_match.nil?
```

Comment:
Oh!  I just realized that we harcoded "Aarons" in here.  I added another success criteria to the list. 😀